sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/sas/AccountSasSignatureValues.java

```diff
@@ -292,6 +312,7 @@ private String stringToSign(final StorageSharedKeyCredential storageSharedKeyCre
             this.sasIpRange == null ? "" : this.sasIpRange.toString(),
             this.protocol == null ? "" : this.protocol.toString(),
             VERSION,
+            this.encryptionScope == null ? "" : this.encryptionScope,
```

Comment:
we should version string to sign.

Reply:
We don't have something like BlobServiceVersion available in the common package to compare against. I can create a similar enum for common, or I can drop a string literal in the conditional statement. Any preference?

Reply:
inline string should be fine.

Reply:
Cool. Then this is done.